packages/schematics/angular/guard/index.ts

```diff
@@ -37,13 +36,14 @@ export default function (options: GuardOptions): Rule {
 
     let implementations = '';
```

Comment:
https://github.com/angular/angular-cli/blob/9b3d50804d6781f82658ac5f887259932ad29cd6/packages/schematics/angular/guard/index.ts#L33-L35

Can we removed as a default is always assigned when it's undefined.